test/plugins/features/movie/index.test.js

```diff
@@ -20,4 +21,28 @@ describe('movies integration', () => {
 
   });
 
+  describe('list', () => {
+
+    const movie = { title: 'foo', release_year: 2016 };
+
+    beforeEach(() => {
+      return Knex('movies').insert(movie);
+    });
+
+    it('lists all movie', () => {
+      return Movies.inject({
+        url: '/movies',
+        method: 'GET',
+        payload: {}
```

Comment:
Since this is an empty object, you can just remove it.

Also a side note, when dealing with `GET`s, you don't pass in a `payload`, you just append any query params you want to test to the URL (e.g. `url: '/movies?release_year=2016'`).